src/moonlink/src/storage/compaction/tests.rs

```diff
@@ -121,6 +124,7 @@ async fn test_data_file_compaction_2() {
     // Create data file and file indices.
     let temp_dir = tempfile::tempdir().unwrap();
     let object_storage_cache = ObjectStorageCache::default_for_test(&temp_dir);
+    let cache: Arc<dyn CacheTrait> = Arc::new(object_storage_cache.clone());
```

Comment:
let's use `object_storage_cache` for naming consistency; there's also ongoing effort on metadata caching.